src/isPalindrome.js

```diff
@@ -19,7 +19,14 @@
  * @return {boolean}
  */
 function isPalindrome(str) {
-  // write code here
+  const regExp = /[^a-z0-9]+/g;
+  const newStr = str.toLowerCase().replace(regExp, '');
```

Comment:
what does `new` mean?
```suggestion
  const replacedString = str.toLowerCase().replace(regExp, '');
```